test/models/name_test.rb

```diff
@@ -2195,6 +2195,36 @@ def test_changing_author_of_autonym
     assert_equal("Ach.", name.author)
   end
 
+  def test_short_display_name
+    # Name without author
+    assert_equal("**__Russula brevipes__**",
+                 names(:russula_brevipes_no_author).short_display_name)
+
+    # Name with author
+    assert_equal("**__Russula brevipes__**",
+                 names(:russula_brevipes_author_notes).short_display_name)
+
+    # Autonym with author
+    autonym = Name.create!(
+      text_name: "Russula sect. Russula",
+      display_name: "**__Russula__** Pers. sect. **__Russula__**",
+      author: "Pers.",
+      rank: :Section,
+      deprecated: false, correct_spelling: nil,
+      user: users(:rolf)
+    )
+    assert_equal("**__Russula__** sect. **__Russula__**",
+                 autonym.short_display_name)
+
+    # group without author
+    assert_equal(names(:unauthored_group).display_name,
+                 names(:unauthored_group).short_display_name)
+
+    # group with author
```

Comment:
Groups are different from other Ranks because the author (if any) does not appear immediately after formatting markup, but after an unformatted "Group"/"Clade"/etc.